<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1887</width>
    <height>989</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,1,0,0">
    <item row="2" column="4">
     <widget class="QPushButton" name="btn_start">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>140</height>
       </size>
      </property>
      <property name="text">
       <string>Lancer la Roulette</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QListWidget" name="liste_tributs_2">
      <property name="minimumSize">
       <size>
        <width>335</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="spacing">
       <number>15</number>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="3">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item row="2" column="0">
     <widget class="QDial" name="nbilles_dial">
      <property name="maximum">
       <number>48</number>
      </property>
      <property name="value">
       <number>8</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Nombre de billes: 8</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2" colspan="2">
     <widget class="QListWidget" name="liste_tributs_1">
      <property name="minimumSize">
       <size>
        <width>335</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="spacing">
       <number>15</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
